### PR TITLE
Modified index.ejs

### PR DESCRIPTION
**Pull request by mapengbo0314**

### CODE REVIEW

**Comment by mapengbo0314** [comment]
This is very good!

**Comment by mapengbo0314** [comment]
This is good

File: views/pages/index.ejs [hunk 1]
```diff
@@ -13,8 +13,8 @@
     <a href="/" class="lang-logo">
       <img src="/lang-logo.png">
     </a>
-    <h1>Getting Started on Heroku with Node.js</h1>
-    <p>This is a sample Node application deployed to Heroku. It's a reasonably simple app - but a good foundation for understanding how to get the most out of the Heroku platform.</p>
+    <h1>Getting started on playing Cod warzone</h1>
+    <p>I am indeed trash at warzone</p>
```
Comment (mapengbo111) [inline views/pages/index.ejs:17]: But you are good if you have a DMR right?

**Review by mapengbo111 — CHANGES_REQUESTED** [review]
This IS very good. GOod job PeNgbOO. But nerf the

**Review by mapengbo111 — APPROVED** [review]
Very good, nerf the DMR indeed

**Pull request merged**